opentelemetry/chart/templates/smon.yaml

```diff
@@ -9,6 +9,7 @@ metadata:
   annotations:
   name: opentelemetry-collector-metrics
   labels:
+    plugin: "{{ .Values.openTelemetry.prometheus }}"
```

Comment:
How does openTelemetry.prometheus map to the otel plugin name?

Reply:
at the moment it has to be set in the plugin page.